interfaces/repo.go

```diff
@@ -747,6 +749,8 @@ func (r *Repository) Connections(snapName string) ([]*ConnRef, error) {
 // coreSnapName returns the name of the core snap if one exists
 func (r *Repository) guessCoreSnapName() (string, error) {
```

Comment:
This function name and its doc needs fixing. This is not about a core snap anymore.

guessSystemSnapName perhaps?

Reply:
+1